examples/nlp/language_modeling/megatron_gpt_eval.py

```diff
@@ -265,9 +267,10 @@ def main(cfg) -> None:
 
     # Second method of running text generation, call trainer.predict
     ds = RequestDataSet(OmegaConf.to_container(cfg.prompts))
-    request_dl = DataLoader(dataset=ds, batch_size=2)
+    request_dl = DataLoader(dataset=ds, batch_size=1)
```

Comment:
is this change necessary?

Reply:
> looks good overall. why we need truncate_prompt_length argument?

If someone want's to truncate their prompt, like with p-tuning truncating the context. but this is not a deal breaker, so your call if you want to remove it

Reply:
ok i can revert the bs